src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -1668,9 +1668,12 @@
   - name: --if-match
     type: string
     short-summary: The value provided will be compared to the ETag of the node pool, if it matches the operation will proceed. If it does not match, the request will be rejected to prevent accidental overwrites. This must not be specified when creating a new agentpool.
+  - name: --ignore-pdb
```

Comment:
the linter CI failed as you added another option form (`-i`) for this parameter, so you'll need to change this help info to one of the followings (I'm not sure which order the CI rule prefers, but it only considers one to be correct)
```suggestion
  - name: --ignore-pdb -i
```
```suggestion
  - name: -i --ignore-pdb
```

Reply:
Thanks for the suggestion. Hope it will work. 
If it works, the git warning is somehow misleading and it should indicate one option is missed but not ignore-pdb is invalid.